dd-sdk-android/src/test/kotlin/com/datadog/android/utils/forge/VitalInfoForgeryFactory.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.utils.forge
+
+import com.datadog.android.rum.internal.vitals.VitalInfo
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.ForgeryFactory
+
+internal class VitalInfoForgeryFactory : ForgeryFactory<VitalInfo> {
+    override fun getForgery(forge: Forge): VitalInfo {
+        val data = forge.aList { aDouble(-65536.0, 65536.0) }
```

Comment:
why those numbers in particular ? (-65536. 65536)

Reply:
Because otherwise the values generated will be at opposite ends of the Double spectrum and generate `NaN` on any calculation